decidim-core/config/locales/en.yml

```diff
@@ -276,6 +276,7 @@ en:
   errors:
     messages:
       content_type_whitelist_error: The file type is not valid
+      cycle_detected: a scope descendant can't be its parent
```

Comment:
shouldn't this be the other way around - `a scope's parent can't be one of its descendants`? Why are not hiding them though?